README.md

```diff
@@ -41,8 +41,8 @@ Function that accepts a `Request` object and returns:
 
 This is used if there is no `rate` function defined in the route plugin [settings](#custom-rate).
 
-##### `requestAPIKey`
-A function that returns an api_key given a `request` object
+##### `rateLimitKey`
+A function that returns a key for an account, given a `request` object. This can be any differentiating value for each user, such as an API Key, IP Address, etc
```

Comment:
A function that returns a key for a given `request`. This can be any differentiating value in each request such as API key, IP address, etc.

Not all request might be associated with an account in the general case.